src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString
+    .split(';')
+    .map((str) => str.split(':'))
+    .map((str) => str.map((value) => value.trim()))
+    .map((str) => str.filter((value) => value !== ''))
+    .filter((str) => str.length !== 0)
+    .reduce((object, item) => {
+      object[item[0]] = item[1];
+
+      return object;
```

Comment:
The use of `reduce` to construct the object is appropriate. However, the parameter name `object` is not ideal as per the checklist. Consider using a more descriptive name, such as `styleObj` or `cssObject`.

Reply:
Looks like you need to rename argument name to increase readability of code